usageDataList.md

```diff
@@ -0,0 +1,15 @@
+## [Dependency Analytics](https://marketplace.visualstudio.com/items?itemName=redhat.fabric8-analytics)
```

Comment:
Point to GH org, because we have openvsx release also.

Reply:
okay

Reply:
done